packages/TorneloScoresheet/src/hooks/appMode/graphicalRecordingState.ts

```diff
@@ -155,7 +155,11 @@ export const makeUseGraphicalRecordingState =
       });
     };
     const goToTextInputFunc = (): void => {};
-    const goToArbiterModeFunc = (): void => {};
+    const goToArbiterModeFunc = (): void => {
+      setAppModeState({
+        mode: AppMode.ArbiterGame,
+      });
+    };
```

Comment:
We need to pass through the graphicalRecordingState's data here, otherwise we will loose it! (Pairing, currentPlayer, moveHistory etc...